src/style.css

```diff
@@ -0,0 +1,68 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
your stars rows will be in a column even without this wrapper 😉
use this class to justify each star row 😁